uiautomationutilities/perf_tool/readme.md

```diff
@@ -0,0 +1,158 @@
+Perf tool walk through recording: https://msit.microsoftstream.com/video/8930a1ff-0400-9887-b080-f1eb55ce8803
```

Comment:
We can improve the formatting of this doc i.e. apply heading styles when appropriate